src/components/Search/Search.js

```diff
@@ -1,11 +1,15 @@
-import React, { useState, useEffect } from 'react';
-import { Route } from 'react-router-dom';
-import PropTypes from 'prop-types';
-
+import React from 'react';
 import './Search.css';
 
 const Search = () => {
-
+  return (
+    <div className='search'>
+      <input type='text' className='search-bar' placeholder='Search'/>
+      <button type='submit' className='search-button'>
+        <i className="fa fa-search"></i>
```

Comment:
Thanks for getting this search feature started for us!
I just looked up the tag for idiomatic text here - excited to see what we can do with this.